src/file.coffee

```diff
@@ -7,7 +7,7 @@ EmitterMixin = require('emissary').Emitter
 fs = require 'fs-plus'
 Grim = require 'grim'
 
-Q = null     # Defer until used
+Q = require 'q'
```

Comment:
I'm primarily developing on node `v0.10.x` and I understand `Promise` comes natively on `0.11.x` - or atom's `v0.13` patch.
Since this is an `npm` package that's supposed to work for other parties and not just atom, and those would probably be using a stable even-numbered node version, MAX: 0.10.x